draft-wang-ppm-differential-privacy.md

```diff
@@ -236,10 +236,61 @@ the gap between actual number of Clients and minimum batch size.
 
 > TODO: Chris P to fill: user or report, given time
 
-## Privacy budget and accounting
+## Privacy budget and accounting {#budget}
 
-> TODO: Junye to fill, Pure ϵ-DP, (ϵ,δ)-approximate DP, (α,τ)-Renyi DP, Zero Concentrated-DP
-> TODO: talk about composability of DP
+There are various types of DP guarantees and budgets that can be enforced.
+Many applications need to query the Client data multiple times, for example:
+
+* Federated machine learning applications are usually composed of multiple
+  iterations.
+
+* {{MJTBCCDFGHJKLLMPPPPPRSWZ22}} describes an interactive approach of building
+  histograms over multiple iterations, and Section 4.3 of
+  {{MJTBCCDFGHJKLLMPPPPPRSWZ22}} describes a way to track Client-side budget
+  when the Client data is queried multiple times.
+
+> TODO: have citations for machine learning
+
+It’s hard for Aggregator(s) to keep track of the privacy budget over time,
+because different Clients can participate in different data collection tasks,
+and only Clients know when their data is queried. Therefore, Clients MUST
+enforce the privacy budget.
+
+There could be multiple ways to compose DP guarantees, based on different
+DP composition theorems. In the various example DP guarantees below,
+we describe the following:
+
+* A formal definition of the DP guarantee.
+
+* Composition theorems that apply to the DP guarantee.
+
+### Pure ϵ-DP, or (ϵ,δ)-approximate DP {#adp}
+
+Pure ϵ-DP was first proposed in {{DMNS06}}, and a formal definition of (ϵ,δ)-DP
+can be found in Definition 2.4 of {{DR14}}.
+
+One can compose multiple (ϵ,δ)-approximate DP guarantees, per Theorem 3.4
+of {{KOV15}}.
```

Comment:
Wrap lines at 80 characters

Reply:
This line is ok, but I changed a couple of other places that have more than 80 characters.